sim/pokemon.ts

```diff
@@ -538,20 +538,20 @@ export class Pokemon {
 			);
 		}
 		if (adjacentOnly) allies = allies.filter(ally => this.battle.isAdjacent(this, ally));
-		return allies.filter(ally => ally && !ally.fainted);
+		return forTargeting ? allies : allies.filter(ally => ally && !ally.fainted);
 	}
 
-	foes(adjacentOnly?: boolean): Pokemon[] {
+	foes(adjacentOnly?: boolean, forTargeting?: boolean): Pokemon[] {
 		let foes = this.side.foe.active;
 		if (this.battle.gameType === 'multi') {
-			const team = this.side.foe.n % 2;
+			const team = +!(this.side.n % 2);
```

Comment:
I don't like casting booleans to strings.

```suggestion
			const team = 1 - (this.side.n % 2);
```

Reply:
Oh that's so much nicer. Wish I'd thought of that.